openhands-api-client/scripts/cloud_api.py

```
@@ -0,0 +1,357 @@
+            issue_number: Issue number
+            comment: Comment text
+            token: GitHub token
+        """
+        url = f'https://api.github.com/repos/{repo}/issues/{issue_number}/comments'
+        headers = {
+            'Authorization': f'token {token}',
+            'Accept': 'application/vnd.github.v3+json',
+        }
+        data = {'body': comment}
+
+        response = requests.post(url, headers=headers, json=data)
```

Ruff (0.14.3)

Probable use of `requests` call without timeout

(S113)